cmd/mungedocs/links.go

```diff
@@ -32,15 +32,15 @@ var (
 	altTextRE = regexp.MustCompile(`(.*)( ".*")`)
 )
 
-// checkLinks assumes fileBytes has links in markdown syntax, and verifies that
-// any relative links actually point to files that exist.
-func checkLinks(filePath string, fileBytes []byte) ([]byte, error) {
-	dir := path.Dir(filePath)
-	errors := []string{}
-
-	output := replaceNonPreformattedRegexp(fileBytes, linkRE, func(in []byte) (out []byte) {
+func doWork(in string, filePath string) (string, error) {
```

Comment:
Given that this is `package main` with a bunch of other mungers, could we have a more descriptive name?